pycontrails/models/cocip/cocip.py

```diff
@@ -985,6 +996,16 @@ def _simulate_contrail_evolution(self) -> None:
         # Intersect with rad dataset
         calc_radiative_properties(self._downwash_contrail, self.params)
 
+        if self.params["preprocess_lowmem"]:
+            return None, None
+        return met, rad
+
+    def _simulate_contrail_evolution(self) -> None:
+        """Simulate contrail evolution."""
+
+        met, rad = self._process_downwash_flight()
```

Comment:
I was hoping I could avoid touching this part of the CoCiP implementation, but it turns out that the code that I extracted to `_process_downwash_flight` sometimes allocates a lot of memory. It's pretty easy to avoid those allocations in `_process_downwash_flight`, but doing that requires some changes to the downselection logic within the main timestepping loop.